test/Libraries/RevitIntegrationTests/RegressionTests.cs

```diff
@@ -268,9 +270,88 @@ public void NoDuplicatePortsOnUINodes()
             File.Delete(Path.Combine(workingDirectory, @".\DupePorts.dyn"));
         }
 
-        // Helper Functions
+      [Test]
+      public void NoUnexpectedDlls_ShouldExist()
+      {
+         // Folder of the test
+         string testAssemblyPath = Assembly.GetExecutingAssembly().Location;
+         string testDirectory = Path.GetDirectoryName(testAssemblyPath);
+
+         // Revit addin build output where the dlls are located
+         string relativeBuildPath = Path.Combine(testDirectory, @"..\..\..\..\..\..\Revit\Debugx64\Addins\DynamoForRevit");
```

Comment:
can we start from the path of current executable? This test does not work if we compile Releasex64 or other configuration